content/_components/ebay/actions.md

```diff
@@ -13,108 +13,84 @@ ComponentVersion: 2.0.2
 
 This call retrieves the details of a specific item, such as description, price, category, all item aspects, condition, return policies, seller feedback and score, shipping options, shipping costs, estimated delivery, and other information the buyer needs to make a purchasing decision.
 
-![Get Item](img/get-item.png)
+{% include img.html max-width="100%" url="img/get-item.png" title="Get Item" %}
 
-*eBay's sandbox environment does not provide correct response for this action. So the code could not be fully tested in development stage. Please be careful when testing this code in production environment. Please contact us if any error occurs.*
+>**Please note:** eBay's sandbox environment does not provide correct response for this action. So the code could not be fully tested in development stage. Please be careful when testing this code in production environment. Please contact us if any error occurs.*
 
 ### Input fields description
 
-__itemId__ - The eBay RESTful identifier of an item. This ID is returned by the Browse and Feed API calls.
+* **itemId** - The eBay RESTful identifier of an item. This ID is returned by the Browse and Feed API calls.
+  * **Format:** RESTful item ID
+  * **For example:** `v1|272394640372|0 `or `v1|272394640372|0`
 
-Format: RESTful item ID
-For example: v1|272394640372|0 or v1|272394640372|0
+* **fieldgroups** - This parameter lets you control what is returned in the response. If you do not set this field, the call returns all the details of the item. Valid values:
 
-__fieldgroups__ - This parameter lets you control what is returned in the response. If you do not set this field, the call returns all the details of the item.
-Valid values:
+  * **PRODUCT** - This adds the additionalImages, additionalProductIdentities, aspectGroups, description, gtins, image, and title product fields to the response, which describe the product associated with the item. See Product for more information about these fields.
```

Comment:
* **PRODUCT** - This adds the `additionalImages`, `additionalProductIdentities`, `aspectGroups`, `description`, `gtins`, `image`, and `title` product fields to the response, which describe the product associated with the item. See `Product` for more information about these fields.